src/loaders/orgUnitLoader.js

```diff
@@ -37,11 +42,8 @@ const orgUnitLoader = async (config) => {
             .catch((error) => {
                 if (error && error.message) {
                     alerts.push({
-                        critical: true,
-                        message: i18n.t('Error: {{message}}', {
-                            message: error.message,
-                            nsSeparator: ';',
-                        }),
+                        code: ERROR_CRITICAL,
+                        message: error.message,
                     })
                 }
             }),
```

Comment:
We should also catch errors from `getOrgUnitLevels(d2)` next line.

Reply:
Let's do this after we have removed d2 (part of the remove schemas effort)